Flow.Launcher.Infrastructure/Image/ImageCache.cs

```diff
@@ -66,8 +66,10 @@ public void Initialization(Dictionary<string, int> usage)
                 {
                     // To delete the images from the data dictionary based on the resizing of the Usage Dictionary.
 
-
-                    foreach (var key in Data.OrderBy(x => x.Value.usage).Take(Data.Count - MaxCached).Select(x => x.Key))
+                    foreach (var key in Data.Where(x => x.Key != Constant.MissingImgIcon)
```

Comment:
if this constant.missingimgicon has low usage i dont see why we shouldnt remove

Reply:
Oh I before use the default icon in the dictionary, so the usage will be extremely high, so I don't want it to affect other resizing. However, since I have make another property for that, that's not significant now.

Reply:
ok, sounds like this change can be reverted?

Reply:
Yes.